forms-flow-forms/api-spec/forms-flow-forms-api-spec.yaml

```diff
@@ -1,814 +1,115 @@
+        _id:
+          type: string
+      required:
+        - tenantKey
+        - title
+        - description
+security:
+  - bearerAuth: []
 paths:
-  /user/login:
+  /role:
     post:
-      tags:
-        - default
-      summary: Login to Form.io
-      requestBody:
-        content:
-          application/json:
-            schema:
-              type: object
-              examples:
-                example-1:
-                  value:
-                    data:
-                      email: test@test.com
-                      password: test
+      summary: ''
+      operationId: post-role
       responses:
         '200':
           description: OK
-          headers:
-            X-Powered-By:
-              schema:
-                type: string
-                example: Express
-            Vary:
-              schema:
-                type: string
-                example: Origin
-            Access-Control-Allow-Origin:
-              schema:
-                type: string
-                example: '*'
-            Access-Control-Expose-Headers:
-              schema:
-                type: string
-                example: x-jwt-token
-            x-jwt-token:
-              schema:
-                type: string
-                example: >-
-                  eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Il9pZCI6IjYyOGYwZWUyMTljZWJiOWNlYTRmMTJiMiJ9LCJmb3JtIjp7Il9pZCI6IjYyOGYwZWUwMTljZWJiOWNlYTRmMTIzZiJ9LCJpYXQiOjE2NTU4OTAyMTUsImV4cCI6MTY1NTkwNDYxNX0.4IEK2N7j9et_VyN2kTDrgb_IUz5ByY9LQgB_Twvs8vE
-            Content-Type:
-              schema:
-                type: string
-                example: application/json; charset=utf-8
-            ETag:
-              schema:
-                type: string
-                example: W/"119-cdOJti4LERpIYeSk+tU22e8fJ6Y"
-            Date:
-              schema:
-                type: string
-                example: Wed, 22 Jun 2022 09:30:15 GMT
-            Connection:
-              schema:
-                type: string
-                example: keep-alive
-            Keep-Alive:
-              schema:
-                type: string
-                example: timeout=5
           content:
             application/json:
               schema:
-                type: object
+                $ref: '#/components/schemas/Role'
               examples:
                 example-1:
                   value:
-                    _id: 628f0ee219cebb9cea4f12b2
-                    form: 628f0ee019cebb9cea4f123f
-                    owner: 628f0ee219cebb9cea4f12b2
-                    roles:
-                      - 628f0edf19cebb9cea4f1226
-                    data:
-                      email: test@test.com
-                    access: []
-                    externalIds: []
-                    created: '2022-05-26T05:23:46.195Z'
-                    modified: '2022-05-26T05:23:46.197Z'
-  /form:
-    post:
-      tags:
-        - default
-      summary: Create a new form
```

Comment:
formio form creation method is also needed. It is used by web API while form creation.
![image](https://user-images.githubusercontent.com/99173163/176593950-3f7f025d-1b17-4354-afc9-3d1334143c4f.png)

Reply:
Sorry, sure will add it back. That was a mistake!

Reply:
@auslin-aot Done